spack-repo/packages/singularity-eos/package.py

```diff
@@ -74,17 +78,17 @@ class SingularityEos(CMakePackage, CudaPackage):
     depends_on("eospac", when="+eospac")
 
     depends_on("ports-of-call@1.4.2:", when="@:1.7.0")
```

Comment:
Do we want to specifically avoid 1.5.1 here since it has the inline issue? Something like
```suggestion
    depends_on("ports-of-call@1.4.2,1.5.2:", when="@:1.7.0")
```